File: spring-backend/src/main/java/com/example/springbackend/dto/FeedbackDTO.java
package com.example.springbackend.dto;

import java.time.LocalDateTime;

public class FeedbackDTO {
    private String id;
    private String userName;
    private String userId;
    private String feedbackText;
    private Integer rating;
    private String tagList;
    private LocalDateTime submittedAt;
}

File: android-kotlin/feature_navigateBar/src/main/java/iss/nus/edu/sg/appfiles/feature_navigatebar/FeedbackDTO.kt
package iss.nus.edu.sg.appfiles.feature_navigatebar


data class FeedbackDTO(
    val id:String?=null,
    val userName: String,
    val userId:String?=null,
    val feedbackText: String,
    val rating: Int,
    val tagList: String,
    val submittedAt: String?=null
)
